docs/installation/server-config/config-reference.rst

```diff
@@ -171,8 +171,12 @@ For an example, see :ref:`restrict-ksql-interactive`.
 listeners
 ---------
 
-Comma-separated list of URIs (including protocol) that the broker will listen on. Specify hostname as ``0.0.0.0`` to bind
-to all interfaces or leave it empty to bind to the default interface. For example:
+Comma-separated list of URIs (including protocol) that the broker will listen on.
```

Comment:
Not sure how this was merged, but this line is in the wrong place. It refers to the `bootstrap.servers` config, not the `listeners` config. 

It seems that this error was made earlier, and simply propagated int his PR.